tasks.py

```diff
@@ -327,13 +352,12 @@ def dev_env(ctx, architecture="amd64", name="kind", protocol=None, frr_volume_di
     else:
         run("kubectl delete po -nmetallb-system --all", echo=True)
 
-        manifests_dir = os.getcwd() + "/manifests"
+        generate_manifests(ctx, bgp_type=bgp_type)
```

Comment:
I am a bit afraid of this, because running dev-env will have a possible side effect of changing the manifests.
I think a cleaner way is to have generate_manifests to take the output path as an optional parameter, and let it generate directly to the temp path. If we do that, we won't even need to tweak the manifest name